test/test_torch.py

```diff
@@ -3305,6 +3306,44 @@ def test_from_buffer(self):
         self.assertEqual(floats.size(), 1)
         self.assertEqual(floats[0], 2.25)
 
+    def test_from_file(self):
+        size = 10000
+        filename = 'testPytorchStorageFromFile'
+        sys = platform.system()
```

Comment:
it's a bad variable name, because it shadows the `sys` module (btw you can just use `sys.platform` for this)